subprojects/plugin-development/src/main/resources/META-INF/gradle-plugins/org.gradle.validate-external-gradle-plugin.properties

```diff
@@ -0,0 +1,17 @@
+#
+# Copyright 2014 the original author or authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+implementation-class=org.gradle.plugin.devel.plugins.ExternalPluginValidationPlugin
```

Comment:
Should this guy live in an external location? Or among the standard Gradle plugins?

I mean, it's probably fine to introduce it here, for now, but we should have a way to promote it to a proper plugin users can use themselves.

Reply:
mmm not sure I follow you: this plugin is already public, right?